near-sdk/src/environment/env.rs

```diff
@@ -152,7 +153,7 @@ pub fn signer_account_id() -> AccountId {
 
 /// The public key of the account that did the signing.
 pub fn signer_account_pk() -> PublicKey {
-    method_into_register!(signer_account_pk)
+    PublicKey::try_from(method_into_register!(signer_account_pk)).unwrap()
```

Comment:
This might want to do `.unwrap_or_else(|_| unreachable())`, as `.unwrap()` includes formatting machinery and associated bloat.

Reply:
That's a good catch 👍. I've also noticed that there's a lot of similar `unwrap`s in `env.rs`. We can evaluate those in a separate issue/PR but I'll change this one to make use of `unwrap_or_else` for now